CHANGELOG.md

```diff
@@ -1,3 +1,172 @@
+
+# 2.12.1
+
+* Fixed: Carbon fails to import multiple times in the same session.
+* Fixed: Carbon fails when used as a nested module and Carbon is loaded globally or by nested in another module.
+
+
+# 2.12.0

```

Comment:
Version headings are incorrect, this should be `2.11.1`

Reply:
Yeah. There was some versioning confusion. 😆